introduction.md

```diff
@@ -2,3 +2,13 @@
 I am Thorgeir Marthinussen
 
 I really like all the Code School paths, except iOS since I don't have any Apple devices
+
+## My deadly skills
+* Java
+* C#
+* JavaScript
+* PHP
+* HTML5
+* CSS
+* Ruby
+* Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?